.circleci/config.yml

```diff
@@ -79,11 +79,17 @@ jobs:
           command: |
             . .venv/bin/activate
             ruff format . --check
+            ruff check .
       - run:
-          name: CurlyLint
+          name: Pip Check
           command: |
             . .venv/bin/activate
-            curlylint wcivf/**/*.html
+            pip check
+      - run:
+          name: Djhtml
+          command: |
+            . .venv/bin/activate
+            git ls-files '*.html' | xargs djhtml --check
```

Comment:
Exact same comment as https://github.com/DemocracyClub/EveryElection/pull/2210#discussion_r1718178871 for the same reasons